metrics/share-of-voice.md

```diff
@@ -0,0 +1,56 @@
+# Share of Voice
+
+Can we communicate the value of our community as a comparison to something of value by our supporting business?
+
+## Description
+
+Ratio of conversation occurring by or about our community in comparison to some baseline. Also described as: Visibility, chatter, social media presence, top-of-funnel impact.
+
+## Objectives
+
+The metric supports the perception of continued value from community work in order to continue to secure funding. It highlights the value driven by community work by comparing it to something already valued by the organization. 
+
+## Implementation
+
+### Prerequisites
```

Comment:
I don't think we need this "subheading".

1. We would make it bold if we wanted to keep it because it is not one of the headings in our [metrics-template](https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md)
2. The next sentence clearly states that it describes prerequisites.
```suggestion
```